desktop/apps/auction/routes.js

```diff
@@ -37,22 +37,14 @@ export async function index (req, res, next) {
   const auctionArticles = new Articles(articles)
 
   try {
-    const html = makeTemplate('index.jade', {
-      basePath: res.app.get('views'),
-      locals: {
-        ...res.locals,
```

Comment:
BTW, I think we determined this was a red herring and the issues was in-fact [template caching](https://github.com/damassi/force/blob/fix-sd-issue/desktop/components/react/utils/template_renderer.js#L39) and not locals/sharify related.